Add contributing-guides/alias-pages.md (#5988)
tldr-pages/tldr: contributing-guides/alias-pages.md

# Alias pages

> To indicate that a command links to another command, we use alias pages.
> This file contains a list of all translations of an alias page template.
> The templates can be chnaged in cases where it makes sense.

Reviewer: `In order to document a command which is an alias of another command, you can write an alias page.` as an introductory sentence is more appealing to me—thoughts? 
```suggestion
In order to document a command which is an alias of another command, you can
write an alias page. This file contains a list of all translations of the alias
page template decided upon in
[#5368](https://github.com/tldr-pages/tldr/pull/5368).
The templates can be changed when necessary.
```

# Alias pages

In order to document a command which is an alias of another command, you can
write an alias page. This file contains a list of all translations of the alias
page template decided upon in
[#5368](https://github.com/tldr-pages/tldr/pull/5368).
The templates can be changed when necessary.

[en](#en) •
[bs](#bs) •
[da](#da) •
[de](#de) •
[es](#es) •
[fa](#fa) •
[fr](#fr) •
[hi](#hi) •
[id](#id) •
[it](#it) •
[ja](#ja) •
[ko](#ko) •
[ml](#ml) •
[nl](#nl) •
[no](#no) •
[pl](#pl) •
[pt_BR](#pt_br) •
[pt_PT](#pt_pt) •
[ru](#ru) •
[sh](#sh) •
[sv](#sv) •
[ta](#ta) •
[th](#th) •
[tr](#tr) •
[zh](#zh) •
[zh_TW](#zh_tw)

### en

```Markdown
# example

> This command is an alias of `example`.

- View documentation for the original command:

`tldr example`
```

### bs

Not translated yet.

### da

Not translated yet.

### de

```Markdown
# beispiel

> Dieser Befehl ist ein Alias von `beispiel`.

- Zeige die Dokumentation für den originalen Befehl an:

`tldr beispiel`
```

### es

Not translated yet.

### fa

Not translated yet.

### fr

> Not translated yet.

Reviewer 2: ```suggestion
# exemple

> Cette commande est un alias de `exemple`.

- Voir la documentation de la commande originale :

`tldr exemple`
```
```

# Alias pages

In order to document a command which is an alias of another command, you can
write an alias page. This file contains a list of all translations of the alias
page template decided upon in
[#5368](https://github.com/tldr-pages/tldr/pull/5368).
The templates can be changed when necessary.

[en](#en) •
[bs](#bs) •
[da](#da) •
[de](#de) •
[es](#es) •
[fa](#fa) •
[fr](#fr) •
[hi](#hi) •
[id](#id) •
[it](#it) •
[ja](#ja) •
[ko](#ko) •
[ml](#ml) •
[nl](#nl) •
[no](#no) •
[pl](#pl) •
[pt_BR](#pt_br) •
[pt_PT](#pt_pt) •
[ru](#ru) •
[sh](#sh) •
[sv](#sv) •
[ta](#ta) •
[th](#th) •
[tr](#tr) •
[zh](#zh) •
[zh_TW](#zh_tw)

### en

```Markdown
# example

> This command is an alias of `example`.

- View documentation for the original command:

`tldr example`

> ```

Reviewer: Hi @2KAbhishek! Yes, can we have a freshly crafted Hindi translation for this snippet (between the \`\`\`)?

Reviewer 3: I'm on it!

Reviewer: Just send a message right here...no need to open a PR or anything like that ;)

Reviewer 3: ```
# उदाहरण

> यह आदेश `उदाहरण` का उपनाम है।

- मूल आदेश के लिए दस्तावेज़ देखें:

`tldr उदाहरण`
```

Reviewer 3: here you go @bl-ue

[en](#en) •
[bs](#bs) •
[da](#da) •
[de](#de) •
[es](#es) •
[fa](#fa) •
[fr](#fr) •
[hi](#hi) •
[id](#id) •
[it](#it) •
[ja](#ja) •
[ko](#ko) •
[ml](#ml) •
[nl](#nl) •
[no](#no) •
[pl](#pl) •
[pt_BR](#pt_br) •
[pt_PT](#pt_pt) •
[ru](#ru) •
[sh](#sh) •
[sv](#sv) •
[ta](#ta) •
[th](#th) •
[tr](#tr) •
[zh](#zh) •
[zh_TW](#zh_tw)

### en

```Markdown
# example

> This command is an alias of `example`.

- View documentation for the original command:

`tldr example`
```

### bs

Not translated yet.

### da

Not translated yet.

### de

```Markdown
# beispiel

> Dieser Befehl ist ein Alias von `beispiel`.

- Zeige die Dokumentation für den originalen Befehl an:

`tldr beispiel`
```

### es

```markdown
# ejemplo

> Este comando es un alias de `ejemplo`.

- Ver documentación para el comando original:

`tldr ejemplo`
```

### fa

Not translated yet.

### fr

```markdown
# exemple

> Cette commande est un alias de `exemple`.

- Voir la documentation de la commande originale :

`tldr exemple`
```

### hi

```markdown
# उदाहरण

> यह आदेश `उदाहरण` का उपनाम है।

- मूल आदेश के लिए दस्तावेज़ देखें:

`tldr उदाहरण`
```

### id

Not translated yet.

### it

Not translated yet.

### ja

> Not translated yet.

Reviewer 4: ```suggestion
```Markdown
# サンプル

> このコマンドは `サンプル` のエイリアスです.

- オリジナルのコマンドのドキュメントを表示する:

`tldr サンプル`
```
```

Reviewer: → f41ac298cdf10d2c3b62b37183b79648f8c412e8

page template decided upon in
[#5368](https://github.com/tldr-pages/tldr/pull/5368).
The templates can be changed when necessary.

[en](#en) •
[bs](#bs) •
[da](#da) •
[de](#de) •
[es](#es) •
[fa](#fa) •
[fr](#fr) •
[hi](#hi) •
[id](#id) •
[it](#it) •
[ja](#ja) •
[ko](#ko) •
[ml](#ml) •
[nl](#nl) •
[no](#no) •
[pl](#pl) •
[pt_BR](#pt_br) •
[pt_PT](#pt_pt) •
[ru](#ru) •
[sh](#sh) •
[sv](#sv) •
[ta](#ta) •
[th](#th) •
[tr](#tr) •
[zh](#zh) •
[zh_TW](#zh_tw)

---
### en

```markdown
# example

> This command is an alias of `example`.

- View documentation for the original command:

`tldr example`
```

---
### bs

```markdown
# example

> Ova komanda je alijas za `example`.

- Pogledaj dokumentaciju za izvornu komandu:

`tldr example`
```

---
### da

Not translated yet.

---
### de

```Markdown
# example

> Dieser Befehl ist ein Alias von `example`.

- Zeige die Dokumentation für den originalen Befehl an:

`tldr example`
```

---
### es

```markdown
# example

> Este comando es un alias de `example`.

- Ver documentación para el comando original:

`tldr example`
```

---
### fa

Not translated yet.

---
### fr

```markdown

> # exemple

Reviewer 2: ```suggestion
# example
```

> > Cette commande est un alias de `exemple`.

Reviewer 2: ```suggestion
> Cette commande est un alias de `example`.
```

- Voir la documentation de la commande originale :

> `tldr exemple`

Reviewer 2: ```suggestion
`tldr example`
```

`tldr example`
```

---
### id

```markdown
# example

> Perintah ini merupakan alias dari `example`.

- Menampilkan dokumentasi untuk perintah asli:

`tldr example`
```

---
### it

Not translated yet.

---
### ja

```markdown
# example

> このコマンドは `example` のエイリアスです.

- オリジナルのコマンドのドキュメントを表示する:

`tldr example`
```

---
### ko

Not translated yet.

---
### ml

```markdown
# example

> ഈ കമാൻഡ് `example` എന്നത്തിന്റെ അപരനാമമാണ്.

- യഥാർത്ഥ കമാൻഡിനായി ഡോക്യുമെന്റേഷൻ കാണുക:

`tldr example`
```

---
### nl

```markdown
# example

> Dit commando is een alias van `example`.

- Bekijk de documentatie van het originele commando:

`tldr example`
```

---
### no

Not translated yet.

---
### pl

```markdown
# example

> Te polecenie jest aliasem `example`.

- Zobacz dokumentację orginalnego polecenia:

`tldr example`
```

---
### pt_BR

Not translated yet.

---
### pt_PT

Not translated yet.

---
### ru

Not translated yet.

---
### sh

Not translated yet.

---
### sv

Not translated yet.

---
### ta

```markdown
# example

> இக்கட்டளை `example` கட்டளையின் மற்றொருப் பெயர்.

- அக்கட்டளையின் விளக்கத்தைக் காண:

`tldr example`
```

---
### th

```markdown
# example

> > คำสั่งนี้เป็น alias ของ `example`.

Author: Hey @nsuthison, is there a reason you didn't translate `alias`? I found "นามแฝง" as a possible translation.

Reviewer 5: @marchersimon `นามแฝง` is not quite the correct translation for this case. I try to translate the word `alias` to many Thai words but all seem hard to understand or feel weird compared to use `alias` directly.

Reviewer 5: To give you more context about this word, `นามแฝง` means the name which is not your real name that you'll use when you're undercover or write a book but didn't want to use your real name.

Marchersimon also is a `นามแฝง` as well.

Reviewer 5: As per @arunisaac suggestion, I'll translate it to this

```
# example

> คำสั่งนี้เป็นอีกชื่อหนึ่งของคำสั่ง `example`.

- เรียกดูรายละเอียดสำหรับคำสั่งตัวเต็ม:

`tldr example`
```

@marchersimon

Author: ```suggestion
> คำสั่งนี้เป็นอีกชื่อหนึ่งของคำสั่ง `example`.
```

- Zeige die Dokumentation für den originalen Befehl an:

`tldr example`
```

---
### es

```markdown
# example

> Este comando es un alias de `example`.

- Ver documentación para el comando original:

`tldr example`
```

---
### fa

Not translated yet.

---
### fr

```markdown
# example

> Cette commande est un alias de `example`.

- Voir la documentation de la commande originale :

`tldr example`
```

---
### hi

```markdown
# example

> यह आदेश `example` का उपनाम है।

- मूल आदेश के लिए दस्तावेज़ देखें:

`tldr example`
```

---
### id

```markdown
# example

> Perintah ini merupakan alias dari `example`.

- Menampilkan dokumentasi untuk perintah asli:

`tldr example`
```

---
### it

Not translated yet.

---
### ja

```markdown
# example

> このコマンドは `example` のエイリアスです.

- オリジナルのコマンドのドキュメントを表示する:

`tldr example`
```

---
### ko

Not translated yet.

---
### ml

```markdown
# example

> ഈ കമാൻഡ് `example` എന്നത്തിന്റെ അപരനാമമാണ്.

- യഥാർത്ഥ കമാൻഡിനായി ഡോക്യുമെന്റേഷൻ കാണുക:

`tldr example`
```

---
### nl

```markdown
# example

> Dit commando is een alias van `example`.

- Bekijk de documentatie van het originele commando:

`tldr example`
```

---
### no

```markdown

> # eksempel

Reviewer 6: ```suggestion
# example
```

Author: Totally forgot about that, thanks for noticing

> > Denne kommandoen er et alias for `eksempel`.

Reviewer 6: ```suggestion
> Denne kommandoen er et alias for `example`.
```

- Vis dokumentasjonen for den opprinnelige kommandoen:

> `tldr eksempel`

Reviewer 6: ```suggestion
`tldr example`
```
